site/getting-started/quick-start.md

```
@@ -0,0 +1,210 @@
+        if err != nil {
+            return err
+        }
+
+        ctx.Export("projectId", project.LagoonId)
+        _ = prodEnv
+        return nil
+    })
+}
+```
+
+#### C#
```

markdownlint-cli2 (0.22.1)

[warning] No space inside hashes on closed atx style heading

(MD020, no-missing-space-closed-atx)